### PR TITLE
[RFR] Refactoring: delete copypasta in test_embedded_ansible_tags

### PR DESCRIPTION
**Pull request by psimovec**
test_tag_ansible_playbook[details] failed in latest test run, not fixing the test case in this PR

### CODE REVIEW

**Comment by dajoRH** [comment]
I detected some fixture changes in commit b4eea266a8b9045de4ef3689896e1f28e98b3c31

The *local* fixture **`obj`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_tags.py**
    - *ansible_tag_test_func*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 8d12fb4c65e45cfb2c40b192f42f04f62a90355a

The *local* fixture **`obj`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_tags.py**
    - *ansible_tag_test_func*

The *local* fixture **`ansible_tag_test_func`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_tags.py**
    - *test_ansible_*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

File: cfme/tests/ansible/test_embedded_ansible_tags.py [hunk 1]
```diff
@@ -60,88 +62,33 @@ def _check_tag_place(item, tag_place):
...
+def ansible_tag_test_func(request, obj):
+    test_func = request.getfixturevalue(request.param)
+    return partial(test_func, obj)
 
 
-@pytest.mark.meta(automates=[1526219])
+@pytest.mark.meta(automates=[1526219, 1526217, 1526218])
 @pytest.mark.parametrize('visibility', [True, False], ids=['visible', 'notVisible'])
-def test_tagvis_ansible_credential(credential, check_item_visibility, visibility):
+def test_ansible_(ansible_tag_test_func, visibility):
```
Comment (mshriver) [inline cfme/tests/ansible/test_embedded_ansible_tags.py]: Though its an 'abstract' test function, I think it should have a complete name. `test_ansible_tagging`

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 83ae6741b74d726b1009b14d60e4f51d0d1c0f0c

The *local* fixture **`obj`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_tags.py**
    - *ansible_tag_test_func*

The *local* fixture **`ansible_tag_test_func`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_tags.py**
    - *test_ansible_tagging*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Pull request merged**